plugins/database/influxdb/influxdb.go

```diff
@@ -133,10 +137,10 @@ func attemptRollback(cli influx.Client, username string, rollbackStatements []st
 			}), "", "")
 
 			response, err := cli.Query(q)
-			if err != nil {
-				if response != nil && response.Error() != nil {
-					return err
-				}
+			// err can be nil with response.Error() being not nil, so both need to be handled
+			merr := multierror.Append(err, response.Error())
+			if merr.ErrorOrNil() != nil {
+				return merr.ErrorOrNil()
```

Comment:
Nit: We can `return merr` directly in here since we've nil checked.

Reply:
Done